micromasters/factories.py

```diff
@@ -14,3 +45,27 @@ class UserFactory(DjangoModelFactory):
 
     class Meta:
         model = User
+
+
+class UserSocialAuthFactory(DjangoModelFactory):
+    """Factory for UserSocialAuth"""
+    user = SubFactory(UserFactory)
+    provider = EdxOrgOAuth2.name
+    extra_data = {"access_token": "fooooootoken"}
+    uid = LazyAttribute(lambda social_auth: social_auth.user.username)
```

Comment:
Can you change this to be different from `user.username`? The `_edx` suffix was originally added to help make it obvious if we leaked the Django username through the API instead of using the edX one